Capstone/Detail View/DetailsViewController.m

```diff
@@ -25,27 +27,30 @@ @interface DetailsViewController () <UITableViewDelegate, UITableViewDataSource>
 @end
 
 @implementation DetailsViewController {
-    NSArray *comments;
-    NSMutableArray *likesArray;
+    NSArray<Comment *> *comments;
+    NSMutableArray<LikesRelations *> *likedCollegesFromParse;
+    NSMutableArray<ParseCollege *> *colleges;
+    NSMutableArray<LikesRelations *> *likedRelations;
+    NSMutableArray<ParseCollege *> *allLikedColleges;
```

Comment:
what's the difference between likedCollegesFromParse and allLikedColleges?

Reply:
resolved